asreviewcontrib/dory/feature_extractors/transformer_embeddings.py

```diff
@@ -46,13 +94,24 @@ def __init__(
                 "sentence_transformer",
                 BaseSentenceTransformer(
                     model_name=self.model_name,
-                    normalize=self.normalize,
```

Comment:
If you do something like:

```python
normalize=self.normalize is True or self.normalize == "l2"
```

(The True ensures backward compat)

then you can simplify the rest of the code:

```python
if self.normalize == "minmax":
    steps.append(("normalizer", MinMaxScaler()))
elif self.normalize == "standard":
    steps.append(("normalizer", StandardScaler()))
```

you don't need the l2 from sklearn then anymore.

Reply:
Oke! Good to me.